test/cluster/mzcompose.py

```diff
@@ -4830,3 +4830,40 @@ def gracefully_alter():
             port=6877,
             user="mz_system",
         )
+
+
+def workflow_crash_on_replica_expiration(
+    c: Composition, parser: WorkflowArgumentParser
+) -> None:
+    """
+    Tests that clusterd crashes when a replica is set to expire
+    """
+    c.down(destroy_volumes=True)
+    with c.override(
+        Testdrive(no_reset=True),
+    ):
+        c.up("testdrive", persistent=True)
+        c.up("materialized")
+        c.up("clusterd1")
+        c.sql(
+            """
+            ALTER SYSTEM SET compute_replica_expiration = 10000;
```

Comment:
This is a duration, so can specify `10s` instead. (I'm not sure what unit it uses by default!)